android/src/main/java/com/ryanheise/just_audio/AudioPlayer.java

```diff
@@ -500,7 +505,11 @@ public void seek(final long position, final Result result) {
 	}
 
 	public void dispose() {
-		player.release();
+    if (player != null) {
+      player.release();
+      player = null;
+		}
+
 		buffering = false;
 		transition(PlaybackState.none);
```

Comment:
These should go inside the `if` since the `dispose` code should only be executed once.